spec/features/merchants/items/update_spec.rb

```diff
@@ -32,7 +32,6 @@
           click_button "Submit"
 
           expect(current_path).to eq("/merchants/#{@merchant1.id}/items/#{@item1.id}")
-save_and_open_page
```

Comment:
Good job keeping these out of the push so we don't have to hunt for one later :)